docs/dev/table/tableApi.md

```diff
@@ -381,6 +381,36 @@ Table result = orders
 {% highlight java %}
 Table orders = tableEnv.scan("Orders");
 Table result = orders.distinct();
+{% endhighlight %}
+        <p><b>Note:</b> For streaming queries the required state to compute the query result might grow infinitely depending on the number of distinct fields. Please provide a query configuration with valid retention interval to prevent excessive state size. See <a href="streaming.html">Streaming Concepts</a> for details.</p>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Distinct Aggregation</strong><br>
+        <span class="label label-primary">Streaming</span>
```

Comment:
I agree, I added the labels. Regarding adding the sections towards each individual `Aggregation` I wasn't able to find a clean construct since some of the discussions (UDAGG, built-in) are general and it's pretty messy to replicate those 3 different ways. 

I regenerated the page and it looks pretty obvious since it is within the `aggregation` tab and all necessary information (such as Over aggregate only applies to stream) is pretty much in the same place.

Reply:
Cool, thx